visualization/app/codeCharta/util/algorithm/treeMapLayout/__snapshots__/treeMapGenerator.spec.ts.snap

```diff
@@ -1293,7 +1293,7 @@ Array [
       "mcc": 10,
       "rloc": 70,
     },
-    "color": "#DDDDDD",
+    "color": "#69AE40",
```

Comment:
This somewhat indicates a behavior change to me. Are we certain everything is as it was before?

Reply:
For triple Checking I checked out main and only set `whiteColorBuildings: true` to `false` in [dataMocks.ts](https://github.com/MaibornWolff/codecharta/blob/main/visualization/app/codeCharta/util/dataMocks.ts#L1593) leading to this exact change in snapshot tests